### PR TITLE
Format code using scalafmt

### PR DESCRIPTION
**Pull request by lxwbr**
#### Has the version number been increased?
 - [ ] Yes! (or not but it was intended to not increase it)

### CODE REVIEW

**Comment by joroKr21** [comment]
Should we check it in CI?

File: src/main/scala/io/moia/protos/teleproto/ReaderImpl.scala [hunk 1]
```diff
@@ -366,20 +362,16 @@ class ReaderImpl(val c: blackbox.Context) extends FormatImpl {
...
     * Unrecognized enum values cause a runtime failure.
     *
-    * (protobuf: ProtoEnum) => p match {
-    *   case ProtoEnum.OPTION_1  => PbSuccess(ModelEnum.OPTION_1)
-    *   ...
-    *   case ProtoEnum.OPTION_N  => PbSuccess(ModelEnum.OPTION_N)
-    *   case Unrecognized(other) => PbFailure(s"Enumeration value $other is unrecognized!")
-    * }
+    * (protobuf: ProtoEnum) => p match { case ProtoEnum.OPTION_1 => PbSuccess(ModelEnum.OPTION_1) ... case ProtoEnum.OPTION_N =>
+    * PbSuccess(ModelEnum.OPTION_N) case Unrecognized(other) => PbFailure(s"Enumeration value $other is unrecognized!") }
```
Comment (saeltz) [inline src/main/scala/io/moia/protos/teleproto/ReaderImpl.scala:374]: Same here.

File: src/main/scala/io/moia/protos/teleproto/WriterImpl.scala [hunk 2]
```diff
@@ -226,15 +223,13 @@ class WriterImpl(val c: blackbox.Context) extends FormatImpl {
...
+    * possible.
     *
-    * (p: model.FooOrBar) =>
-    *   if (p.isInstanceOf[model.Foo])
-    *     protobuf.FooOrBar(protobuf.FooOrBar.Value.Foo(transform[model.Foo, protobuf.Foo](value.asInstanceOf[model.Foo])))
-    *   else
-    *     protobuf.FooOrBar(protobuf.FooOrBar.Value.Bar(transform[model.Bar, protobuf.Bar](value.asInstanceOf[model.Bar])))
+    * (p: model.FooOrBar) => if (p.isInstanceOf[model.Foo]) protobuf.FooOrBar(protobuf.FooOrBar.Value.Foo(transform[model.Foo,
+    * protobuf.Foo](value.asInstanceOf[model.Foo]))) else protobuf.FooOrBar(protobuf.FooOrBar.Value.Bar(transform[model.Bar,
+    * protobuf.Bar](value.asInstanceOf[model.Bar])))
```
Comment (saeltz) [inline src/main/scala/io/moia/protos/teleproto/WriterImpl.scala:232]: Should be in a code block.

File: src/main/scala/io/moia/protos/teleproto/WriterImpl.scala [hunk 3]
```diff
@@ -269,17 +264,12 @@ class WriterImpl(val c: blackbox.Context) extends FormatImpl {
...
-    * If there are more options on the model side, the mapping is not possible.
+    * If there are more options on the protobuf side, the mapping is forward compatible. If there are more options on the model side, the
+    * mapping is not possible.
     *
-    * (model: ModelEnum) => p match {
-    *   case ModelEnum.OPTION_1 => ProtoEnum.OPTION_1
-    *   ...
-    *   case ModelEnum.OPTION_N => ProtoEnum.OPTION_N
-    * }
+    * (model: ModelEnum) => p match { case ModelEnum.OPTION_1 => ProtoEnum.OPTION_1 ... case ModelEnum.OPTION_N => ProtoEnum.OPTION_N }
```
Comment (saeltz) [inline src/main/scala/io/moia/protos/teleproto/WriterImpl.scala:272]: Same here.

**Review by saeltz — CHANGES_REQUESTED** [review]

**Comment by saeltz** [comment]
Superseded by #185.

**Pull request closed without merging**